### PR TITLE
Added jobs and gang grades

### PR DESCRIPTION
**Pull request by imgucciwz**
Describe Pull request
Added the option to be able to add the items with job and gangs grades.

If your PR is to fix an issue mention that issue here

Questions (please complete the following information):

Have you personally loaded this code into an updated qbcore project and checked all it's functionality? [yes/no] (Be honest) Yes Does your code fit the style guidelines? [yes/no] Yes Does your PR fit the contribution guidelines? [yes/no] Yes

Describe Pull request
First, make sure you've read and are following the contribution guidelines and style guide and your code reflects that.
Write up a clear and concise description of what your pull request adds or fixes and if it's an added feature explain why you think it should be included in the core.

If your PR is to fix an issue mention that issue here

Questions (please complete the following information):

Have you personally loaded this code into an updated qbcore project and checked all it's functionality? [yes/no] (Be honest)
Does your code fit the style guidelines? [yes/no]
Does your PR fit the contribution guidelines? [yes/no]

### CODE REVIEW

**Review by ChrisNewmanVortex — APPROVED** [review]
You are awesome, thank you so much <3

File: client/main.lua [hunk 1]
```diff
@@ -25,9 +25,9 @@ local function createBlips()
     end
 end
 
-local function tableCheck(val, list)
-    for _, v in ipairs(list) do
-        if v == val then return true end
+local function tableCheck(val, val2, list)
+    for k, v in pairs(list) do
```
Comment (citRaTTV) [inline client/main.lua:29]: My only concern w/ this change is backwards compat. for folks that are updating an existing deploy. Perhaps we could check to see the type of each key to see if it's an integer or a string, and behave appropriately.

**Review by citRaTTV — CHANGES_REQUESTED** [review]

File: client/main.lua [hunk 2]
```diff
@@ -60,11 +60,11 @@ local function openShop(shop, data)
                 local curProduct = products[i]
                 local addProduct = true
 
-                if curProduct.requiredJob and not tableCheck(PlayerData.job.name, curProduct.requiredJob) then
+                if curProduct.requiredJob and not tableCheck(PlayerData.job.name, PlayerData.job.grade.level, curProduct.requiredJob) then
                     addProduct = false
                 end
 
-                if curProduct.requiredGang and not tableCheck(PlayerData.gang.name, curProduct.requiredGang) then
+                if curProduct.requiredGang and not tableCheck(PlayerData.gang.name, PlayerData.job.grade.level, curProduct.requiredGang) then
```
Comment (citRaTTV) [inline client/main.lua:67]: Also, this should be `PlayerData.gang.grade.level`

File: client/main.lua (hunk 1 above)
Comment (ChrisNewmanVortex) [reply client/main.lua:29]: Worked perfectly for me!

**Review by ChrisNewmanVortex — COMMENTED** [review]
Perfecto!

**Review by ChrisNewmanVortex — APPROVED** [review]

**Review by ChrisNewmanVortex — APPROVED** [review]
Works well!

File: client/main.lua (hunk 2 above)
Comment (citRaTTV) [reply client/main.lua:67]: @imgucciwz

**Pull request closed without merging**